charter.md

```diff
@@ -0,0 +1,74 @@
+
+### In scope
+
+- Ensuring quality Kubernetes releases
+  - Defining and staffing release roles to manage the resolution of release blocking criteria
+  - Defining and driving development processes (e.g. merge queues, cherrypicks) and release processes
+    (e.g. burndown meetings, cutting beta releases) with the intent of meeting the release schedule
+  - Managing the creation of release specific artifacts, including:
+    - Code branches
+    - Binary artifacts
+    - Release notes
+- Continually improving release and development processes
+  - Working closely with SIG Contributor Experience to define and build tools to facilitate release process (e.g. dashboards)
+  - Working closely with SIG Testing to determine and implement tests, automation, and labeling required for stable releases
+  - Working with downstream communities responsible for packaging Kubernetes releases
+  - Working with other SIGs to agree upon the responsibilities of their SIG with respect to the release
+  - Defining and collecting metrics related to the release in order to measure progress over each release
+  - Facilitating release retrospectives in conjunction with SIG PM
+- Collaborating with downstream communities which build artifacts from Kubernetes releases
+  - Ensuring the appropriate level of integration with publishing the build artifacts
+
+### Out of scope
+
+#### Support
+
+SIG Release itself is not responsible for end user support or creation of patches for support streams. There are support forums for end users to ask questions and report bugs, subject matter experts in other SIGs triage and address issues and when necessary mark bug fixes for inclusion in a patch release.
+
+## Roles and Organization Management
+
+This SIG adheres to the Roles and Organization Management outlined in [sig-governance] and opts-in to updates and modifications to [sig-governance].
+
+Specifically, the common guidelines (see: [sig-governance]) for continuity of membership within roles in the SIG are followed.
+
+### Deviations from [sig-governance]
+
+- SIG Release does not have top-level SIG Technical Leads; SIG Release Chairs can serve in both an operational and technical capacity
+- SIG Release subprojects have subproject chairs
+
+#### SIG Membership
+
+Additionally, SIG Release has a concept of membership. SIG members can be occasionally called on to assist with decision making, especially as it relates to gathering historical context around existing policies.
+
+While a strict definition of membership has yet to be established, we will initially seed a set of members from the list of all documented former Release Team members and subproject OWNERS.
+
+SIG Release Members should be the first group of community members considered when seeking to fill additional roles across SIG Release proper and SIG Release subprojects, except where explicitly defined in other policy.
```

Comment:
I would be comfortable taking a harder line here. Unless otherwise agreed (release team shadow roles are a notable existing example of this), _all_ official SIG roles may only be filled by SIG members

Reply:
Is there a universal benefit in doing so?  It feels to me that being exclusive in this regard would be the opposite of acting as an inclusive and open community.

Reply:
@calebamiles -- Happy to take a harder line here as well, and I've updated the language to reflect that.

@tpepper -- I agree with Caleb here. There are multiple ingress points to SIG membership, all of which require some active contribution. I don't feel that they're unnecessarily restrictive or exclusive.

Examples:
- Joe serves as a Release Team shadow and subsequently becomes a role lead. At the end of that cycle, he'd officially become a SIG Release member.
- Liz contributes to the Release Engineering tooling and subsequently graduates to the OWNERS file for the release-engineering subproject. Liz would also officially become a SIG Release member at that point.

Both examples show sustained commitment to SIG Release, and I believe that that slightly higher bar is paramount for contributors interested in defining policy and process for the systems, tooling, and teams that help us release Kubernetes.

Reply:
OK I can accept that reasoning.